src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -74,9 +90,11 @@ private static void Main(string[] args)
         // Issues a GET request to the Bridge to determine whether it is alive.
         // A return of 'true' means the Bridge is healthy.  A return of 'false'
         // indicates the Bridge is not healthy, and 'errorMessage' describes the problem.
-        private static bool PingBridge(string host, int port, out string errorMessage)
+        // A healthy response will also populate properties with the current Bridge configuration.
+        private static bool PingBridge(string host, int port, out string errorMessage, out Dictionary<string, string> properties)
```

Comment:
In general I recommend us not to create something until we actually use it. We have seen so many times in the past things were created because they were thought to be useful but never used and caused confusions.

In this case, if you guys feel strongly we will use it, you can keep it. Maybe we can create another overload, so we don't have to change all existing code that calls PingBridge.